schema.sql

```diff
@@ -6,5 +6,19 @@ CREATE TABLE animals(
     escape_attempts integer,
     neutered boolean,
     weight_kg decimal,
-    species TEXT
-);
\ No newline at end of file
+    FOREIGN KEY(species_id) REFERENCES species(id),
+    FOREIGN KEY(owner_id) REFERENCES species(id)
+);
+
+CREATE TABLE owners(
+    id SERIAL PRIMARY KEY,
+    full_name text,
```

Comment:
- [ ] instead, of using `text` as the data type for the full_name column, I'll suggest you use the `varchar(n)` data type so that you can attribute a limit to the the name as it can have limited numbers of characters.